src/Paket.Core/Utils.fs

```diff
@@ -230,7 +230,7 @@ let RunInLockedAccessMode(rootFolder,action) =
     with
     | exn ->
             releaseLock()
-            raise exn
+            reraise()
```

Comment:
``raise exn`` resulted in losing the stacktrace, hence verbose mode in case of exception always showed a newly generated stacktrace